source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/new/display.tsx

```diff
@@ -1,27 +1,29 @@
 import * as React from 'react';
 import { Card, Form, Checkbox } from 'semantic-ui-react';
-import { Link } from 'react-router-dom';
+import { Link, RouteComponentProps } from 'react-router-dom';
```

Comment:
I feel like this library has 542 different prop exports that we can use / use incorrectly. no idea what one is correct

Reply:
https://stackoverflow.com/questions/49342390/typescript-how-to-add-type-check-for-history-object-in-react